.github/actions/run-tests/action.yml

```diff
@@ -0,0 +1,21 @@
+name: "Run tests"
+description: "Run tests and archive test results for subsequent publication"
+inputs:
+  command:
+    required: true
+    description: "The shell command to run tests"
+runs:
+  using: "composite"
+  steps:
+    - name: Run Tests
+      shell: bash
+      run: ${{ inputs.command }}
```

Comment:
is this action used? (performance and integration tests seem to have their own implementation)

Reply:
I don't see it being called either. This is coming from upstream. I could delete it, but not exactly sure how much we can deviate from the main repo.